### DISCUSSION TITLE
藏文识别

### DISCUSSION THREAD

**Discussion by hzn666** [PaddlePaddle/PaddleOCR#12269] [Ideas & Features]
需求描述：支持藏文模型
需求场景： 覆盖藏文的OCR
潜在解决方案：调取藏文数据集进行训练。
藏文数据集：https://www.heywhale.com/mw/dataset/5bfe734a954d6e0010683839

**Comment by changdazhou** [comment]
已将您的需求记录，我们会评估后续是否支持哈
